src/sql/workbench/contrib/editData/browser/editDataGridPanel.ts

```diff
@@ -1131,6 +1151,11 @@ export class EditDataGridPanel extends GridParentComponent {
 			// Since we need to return a string here, we are using calling a function instead of event emitter like other events handlers
 			return this.onBeforeAppendCell ? this.onBeforeAppendCell(args.row, args.cell) : undefined;
 		});
+		this.table.grid.onKeyDown.subscribe((e, args) => {
```

Comment:
We already hook up the key listener on line 134 - why do you need to add another one here?

Reply:
For some reason, pressing enter does not work with the key listener there, it is not captured at all. My guess is that SlickGrid's native OnKeyDown swallowed the event and moved the focus down (since the enter key has a known use for base slickgrid), so you have to wire the event in the grid itself rather than the container.

Reply:
Not necessary, I'm removing it.